### PR TITLE
throw NotEnoughBalanceInParentAccount if parent can't afford fees

### PR DESCRIPTION
**Pull request by juan-cortes**
> Requires https://github.com/LedgerHQ/ledgerjs/pull/420 and a bump

For Tezos child accounts, check if the parent account can afford the transaction fees and throw the appropriate error if it can't.

### CODE REVIEW

**Comment by juan-cortes** [comment]
Once again, this was supposed to be a draft.

File: src/families/tezos/bridge/libcore.js [hunk 1]
```diff
@@ -132,6 +132,10 @@ const getTransactionStatus = async (a, t) => {
     );
   }
 
+  if (estimatedFees.gt(a.balance)) {
```
Comment (gre) [inline src/families/tezos/bridge/libcore.js]: to me we should do that if:
- there is no errors.amount already
- it's a "subAccuont" transaction 🤔

Comment (juan-cortes) [reply src/families/tezos/bridge/libcore.js]: current version should do that

**Comment by gre** [comment]
I can't find this PR it keeps saying #404

**Pull request merged**